### DISCUSSION TITLE
Custom Blocks in all sprites.

### DISCUSSION THREAD

**Discussion by 3dModel101** [ScratchAddons/ScratchAddons#4271] [Ideas]
I think that when making a custom block, there should be a think like variables that says "for all sprites" or "for this sprite only". It would help a lot, especially when trying to run animations in turbo mode, like wait 0.01 seconds, next costume, and still having your project run faster if you know what I mean. I'm also really excited for the new debug menu update to come out on turbowarp.

**Comment by LilSefGaming** [comment]
I support, you could Make a block in that category!
